test/core/memory_usage/memory_usage_test.cc

```diff
@@ -72,26 +79,55 @@ int main(int argc, char** argv) {
   } else {
     strcpy(root, ".");
   }
-  /* start the server */
-  Subprocess svr({absl::StrCat(root, "/memory_usage_server",
-                               gpr_subprocess_binary_extension()),
-                  "--bind", grpc_core::JoinHostPort("::", port), "--nosecure",
-                  absl::StrCat("--minstack=", absl::GetFlag(FLAGS_minstack))});
-
-  /* start the client */
-  Subprocess cli({absl::StrCat(root, "/memory_usage_client",
-                               gpr_subprocess_binary_extension()),
-                  "--target", grpc_core::JoinHostPort("127.0.0.1", port),
-                  absl::StrCat("--warmup=", absl::GetFlag(FLAGS_warmup)),
-                  absl::StrCat("--benchmark=", absl::GetFlag(FLAGS_benchmark)),
-                  absl::StrCat("--minstack=", absl::GetFlag(FLAGS_minstack))});
-
-  /* wait for completion */
-  if ((status = cli.Join()) != 0) {
-    printf("client failed with: %d", status);
-    return 1;
+
+  /* Set configurations */
+  if (absl::GetFlag(FLAGS_size) == 0) absl::SetFlag(&FLAGS_size, 50000);
```

Comment:
How about we remove this line and just say that if size flag is not set, we default to 50000 (which is the default of the flag anyway)?

Reply:
Alright, seems reasonable now that we changed the default of the flag to 50000, I'll update the documentation accordingly.